src/server/pachyderm_test.go

```diff
@@ -5444,6 +5445,33 @@ func TestService(t *testing.T) {
 		return nil
 	}, backoff.NewTestingBackOff()))
 
+	clientAddr := c.GetAddress()
+	host, _, err := net.SplitHostPort(clientAddr)
+	port, ok := os.LookupEnv("PACHD_SERVICE_PORT_API_HTTP_PORT")
+	if !ok {
+		port = "30652" // default NodePort port for Pachd's HTTP API
```

Comment:
Not sure if this port is exported, but if so I'd prefer we use that and not hardcode any ports in the tests.

Reply:
It isn't exported afaict, this is just copy pasted from the other tests of the http server. I agree that ideally we would always use names for these ports, but it doesn't seem quite worth it to dive in and clean them all up right now.

Reply:
Alright. If we do change them, the tests will fail, which seems sufficient.